lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -1364,6 +1369,18 @@ ChangeResult TypeAnalyzer::visitAtenMatmulOp(
   return getLatticeElement(op->getResult(0)).join(knowledge);
 }
 
+ChangeResult TypeAnalyzer::visitAtenMeanOp(
```

Comment:
You can use

https://github.com/llvm/torch-mlir/blob/ca8b73a65ee1f47ce1dd11443bf277e74e6e6528/lib/Dialect/Torch/Transforms/RefineTypes.cpp#L926